crates/std_detect/tests/cpu-detection.rs

```diff
@@ -45,6 +45,7 @@ fn aarch64_linux() {
     println!("rdm: {}", is_aarch64_feature_detected!("rdm"));
     println!("rcpc: {}", is_aarch64_feature_detected!("rcpc"));
     println!("dotprod: {}", is_aarch64_feature_detected!("dotprod"));
+    println!("tme: {}", is_aarch64_feature_detected!("tme"));
```

Comment:
You need to implement detection for this feature. Do a search for `dotprod` to see how it is done.

Reply:
Updated with feature detection, didn't find the feature tbh. Even instructions and signatures are the same. https://github.com/llvm/llvm-project/commit/a36d31478c182903523e04eb271bbf102bfab2cc#diff-3674dac5c4241890431fece8594c3143R25-R34

What am I doing wrong? (I still suspect from qemu, checked source code, it doesn't exist there yet. Might be wrong.)

Reply:
The current feature detection is fine. Linux doesn't have the TME flag in /proc/cpuinfo or AT_HWCAPS yet, so there's nothing we can do about that for now.

Reply:
Yes, when it enters to kernel I will add here.

Reply:
Addressed your comments. Thanks for the help! @Amanieu